PROFILER.md

```diff
@@ -9,21 +9,25 @@ which inspects the state of all threads to determine how long they have been
 running for. This does not require any code to be modified to support profiling
 so that any code which runs will be profiled automatically.
 
-The profiler can be enabled by setting the following:
+## Enabling Profiling
 
- * `com.iopipe.plugin.profiler` to `true`
- * `IOPIPE_PROFILER_ENABLE` to `true`
+Set the system property `com.iopipe.plugin.profiler` to `true`
```

Comment:
This should be set the environment variable (or mention both), you can't set system properties on Lambda, the system property _is_ available in the system should someone be running some sort of alternate setup